stable/magic-namespace/README.md

```diff
@@ -140,6 +140,13 @@ reference the default `values.yaml` to understand further options.
 | `tiller.role.type` | Identify the name of the `Role` or `ClusterRole` that will be referenced in the role binding for Tiller's service account. There is seldom any reason to override this. | `admin` |
 | `tiller.includeService` | This deploys a service resource for Tiller. This is not generally needed. Please understand the security implications of this before overriding the default. | `false` |
 | `tiller.onlyListenOnLocalhost` | This prevents Tiller from binding to `0.0.0.0`. This is generally advisable to close known Tiller-based attack vectors. Please understand the security implications of this before overriding the default. | `true` |
+| `tls.enabled` | Whether to enable TLS encryption between Helm and Tiller. | `false` |
+| `tls.verify` | Whether to verify a remote certificate. | `true` |
+| `tls.certPath` | Path to mount certificates into the container. | `/etc/certs` |
+| `tls.secretName` | Provide TLS certs via an existing secret with Path to mount certificates into the container. Must include ca.crt, tls.crt and tls.key files| `nil` |
```

Comment:
Is this option to use an existing secret mutually exclusive with the `tls.ca`, `tls.cert`, and `tols.key` options below?

If so, I feel like the documentation should make this clear.

Reply:
Actually... now that I am thinking about it more, I'm not sure that the option to use an _existing_ secret makes much sense. The point of this chart is to _create_ and manage namespaces. This being the case, there cannot _be_ an existing secret in the namespace that you use this chart to manage.

Reply:
Hi @krancour , thanks for your review! It made me notice that I was specifying the options as `tls.foo` rather than `tiller.tls.foo`.  In relation to the custom secret, I too thought this when I assembled the PR, although the namespace creation is dependant on specifying _namespace_ i.e. `{{- if hasKey .Values "namespace" }}`, so it's feasible to use an existing namespace, with an existing secret.

Reply:
@acaire good point. I forgot I'd made that optional to support the edge case where the namespace already exists.